src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/SourceControlManagement.java

```diff
@@ -200,16 +202,97 @@ public void verifyPipelineList(int size) {
     );
   }
 
+  @Then("Verify local pipeline list size {int}")
```

Comment:
Where is this being used ? Cannot seem to find any refernce

Reply:
removed.